include/beast/http/concepts.hpp

```diff
@@ -116,31 +163,97 @@ class is_Parser
 
 } // detail
 
-/// Determine if `T` meets the requirements of `Body`.
+/// Determine if `T` meets the requirements of @b Body.
 template<class T>
 #if GENERATING_DOCS
 struct is_Body : std::integral_constant<bool, ...>{};
 #else
-using is_Body = typename detail::is_Body<T>::type;
+using is_Body = detail::has_value_type<T>;
 #endif
 
-/// Determine if `T` meets the requirements of `ReadableBody`.
-template<class T>
+/** Determine if a @b Body has a nested type `reader`.
+
+    @tparam T The type to check, which must meet the
+    requirements of @b Body.
```

Comment:
Yeah, that's unfortunately a bug in the xsl. See:
https://github.com/vinniefalco/docca/issues/8